app/build.gradle

```diff
@@ -39,24 +38,24 @@ android {
 
 dependencies {
     compile 'com.squareup:otto:1.3.8'
-    compile 'org.osmdroid:osmdroid-android:5.4:release@aar'
+    compile 'org.osmdroid:osmdroid-android:5.6.2'
     compile 'com.squareup.picasso:picasso:2.5.2'
-    compile 'com.jakewharton.picasso:picasso2-okhttp3-downloader:1.0.2'
+    compile 'com.jakewharton.picasso:picasso2-okhttp3-downloader:1.1.0'
     compile "com.android.support:support-v4:$support_version"
     compile "com.android.support:appcompat-v7:$support_version"
     compile "com.android.support:support-annotations:$support_version"
     compile "com.android.support:design:$support_version"
 
-    compile 'com.squareup.okhttp3:okhttp:3.4.1'
+    compile 'com.squareup.okhttp3:okhttp:3.5.0'
     compile "com.jakewharton:butterknife:$butterknife_version"
-    apt "com.jakewharton:butterknife-compiler:$butterknife_version"
+    annotationProcessor "com.jakewharton:butterknife-compiler:$butterknife_version"
 
     compile 'org.ligi:AXT:0.35'
     compile 'info.metadude.android:typed-preferences:2.1.0'
     compile "com.google.dagger:dagger:$dagger_version"
 
     provided 'javax.annotation:jsr250-api:1.0'
-    apt "com.google.dagger:dagger-compiler:$dagger_version"
+    annotationProcessor "com.google.dagger:dagger-compiler:$dagger_version"
```

Comment:
Yes. @hvisser announced it [here](http://www.littlerobots.nl/blog/Whats-next-for-android-apt/).